templates/worker-statefulset.yaml

```
@@ -60,6 +60,10 @@ spec:
           image: "{{ .Values.image }}:{{ .Values.imageTag }}"
           {{- end }}
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+{{- if .Values.worker.init-resources }}
```

The `init-resources` was not added to values.yaml. Could you add it there please?

Alternatively, you could also set it to `worker.resources` instead. I think that would be suitable.
```suggestion
{{- if .Values.worker.resources }}
```

---

nit: could you indent the `if` statement so it lines up with the above if-statements. Thanks!